src/ILCompiler.Compiler/src/Compiler/DefFileWriter.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+
+using Internal.TypeSystem;
+using Internal.TypeSystem.Ecma;
+
+namespace ILCompiler
+{
+    public class DefFileWriter
```

Comment:
It is called ExportsFile everywhere else in this change. This should be called ExportsFileWriter to match.